test/unsafe_methods_test.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "minitest/autorun"
+require "rgeo/shapefile"
+
+class UnsafeMethodsTest < Minitest::Test
+  def test_unsafe_methods_true
+    _open_shapefile("test", true) do |file_|
+      file_.next # Does not raise an error
```

Comment:
This appears to be the Minitest philosophy but I'm not super familiar with it so please tell me if this is off or isn't testing enough

Reply:
maybe we could assert that the resultant geometry is in fact invalid as a sanity check here, but I think this is fine as well

Reply:
Thanks for reviewing! Good suggestion - added!